gap/PolygonalComplexes/modification.gd

```diff
@@ -241,8 +271,25 @@ DeclareOperation( "SplitVertexNC", [IsPolygonalComplex, IsPosInt, IsList] );
 #! same position in <A>oldPath</A>.
 #!
 #! TODO explain better
-#!
-#! TODO many, many examples
+#! 
+#! For example consider the following polygonal complex:
+#! @BeginExampleSession
+#! gap> complex:=PolygonalComplexByDownwardIncidence([[1,2],[2,3],[1,3],[1,4],
+#! > [3,4],[3,5],[5,6],[3,6],[3,7],[6,7]],[[1,2,3],[3,4,5],[6,7,8],[8,9,10]]);;
+#! @EndExampleSession
+#! <Alt Only="TikZ">
+#!   \input{Image_SplitEdgePath.tex}
```

Comment:
I think that the labels in the image are wrong. Since the shared vertex should be 3 and not 2.